translate/misc/selector.py

```diff
@@ -125,7 +125,7 @@ def add(self, path, method_dict=None, prefix=None, **http_methods):
         
         Prefix will override self.prefix for this mapping.
         """
-        # Thanks to S�bastien Pierre 
+        # Thanks to Sébastien Pierre 
```

Comment:
```suggestion
        # Thanks to Sébastien Pierre
```
Whitespace issue? This should not be on the diff :-)